File: 20220831/모범답안/20220831_05.js/DuplicateOdd.py
def solution(arr):
    """
    :param arr: int[]
    :return: int[]
    """

    result = []

    i = 0
    while len(result) < len(arr):
        element = arr[i]
        result.append(element)
        if element % 2 == 1 and len(result) < len(arr):
            result.append(element)

        i += 1

    return result

File: 20220831/모범답안/20220831_05.js/DuplicateOdd.js
/**
 * @param arr {number[]}
 * @returns {number[]}
 */
function solution(arr) {
  const result = []

  let i = 0
  while (result.length < arr.length) {
    const element = arr[i++]
    result.push(element)
    if (element % 2 === 1 && result.length < arr.length) {
      result.push(element)
    }
  }

  return result
}

solution